Task 1/Time conversion/Question.md

```diff
@@ -0,0 +1,19 @@
+Given a time in 12-hour AM/PM format, in a string, convert it to military (24-hour format) time.
```

Comment:
Add headings and subheading in markdown format

Reply:
ok Let me do it